apps/cli/src/commands/environment/create.env.ts

```diff
@@ -0,0 +1,76 @@
+import BaseCommand from '../base.command'
+import Logger from '../../util/logger'
+import {
+  CommandActionData,
+  CommandOption
+} from 'src/types/command/command.types'
+import { text } from '@clack/prompts'
+import { EnvironmentController } from '@keyshade/api-client'
+import {
+  CreateEnvironmentRequest,
+  CreateEnvironmentResponse
+} from '../../../../../packages/api-client/src/types/environment.types'
+export class CreateEnvironment extends BaseCommand {
+  getName(): string {
+    return 'create'
+  }
+
+  getDescription(): string {
+    return 'Create a new environment'
+  }
+
+  getOptions(): CommandOption[] {
+    return []
+  }
+
+  async action({ options, args }: CommandActionData): Promise<void> {
+    const [projectId] = args
+    const { name, description } = await this.parseInput(options)
+
+    if (!projectId) {
+      Logger.error('Project ID is required')
+      return
+    }
+
+    const baseUrl = this.baseUrl
+    const apiKey = this.apiKey
+
+    const environmentData: CreateEnvironmentRequest = {
+      name: name,
+      description: description,
+      projectId: projectId
+    }
+
+    const headers = {
+      baseUrl,
+      apiKey
+    }
+
```

Comment:
We would like to wrap this using a clack spinner. And instead of using `Logger`, we would like to use `intro`, `outro` and other such clack util functions.